gobblin-config-management/gobblin-config-core/src/main/java/gobblin/config/common/impl/SingleLinkedListConfigKeyPath.java

```diff
@@ -0,0 +1,56 @@
+package gobblin.config.common.impl;
+
+import gobblin.config.store.api.ConfigKeyPath;
+
+public class SingleLinkedListConfigKeyPath implements ConfigKeyPath{
+  
+  public static final SingleLinkedListConfigKeyPath ROOT = new SingleLinkedListConfigKeyPath(null, "/");
```

Comment:
This is not correct. The root name is "". "/" is the separator between the root own name and the first child.